app/login/page.tsx

```diff
@@ -50,6 +53,20 @@ const LogIn = () => {
     }
   };
 
+  // error while appwrite updating document
+  // useEffect(() => {
+  const fetchData = async () => {
+    try {
+      const result = await saveUserToDB(userDoc);
```

Comment:
Instead of setting `setUserDoc`, we should just put this `saveUserToDB` right after the login.